src/styles/blocks.scss

```diff
@@ -0,0 +1,41 @@
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  height: 400px;
+  width: 400px;
+  border: 4px solid #{$clock-frame-color};
+  border-radius: 50%;
+}
+  
+.clock__center {
+  height: 20px;
+  width: 20px;
+  border: 2px solid #{$clock-center-color};
+  border-radius: 50%;
+  background-color: #{$clock-center-color};
+  z-index: 1;
+}
+  
+.clock__minute-hand,
+.clock__hour-hand {
+  position: absolute;
+  top: 8px;
+  width: 5px;
+  height: 190px;
+  border: 2px solid #{$minute-hand-color};
+  border-radius: 5px;
+  background-color: #{$minute-hand-color};
+  transform-origin: bottom;
+  animation: rotate 60s linear infinite;
+}
+  
+.clock__hour-hand {
+  top: 48px;
+  width: 5px;
+  height: 150px;
+  border: 2px solid #{$hour-hand-color};
+  background-color: #{$hour-hand-color};
+  animation: rotate 3600s linear infinite;
+}
```

Comment:
it will be more clear to use common class .clock__hand with common styles and modifiers with differences